napari/components/_tests/test_prune_kwargs.py

```diff
@@ -29,7 +28,6 @@
         'labels',
         {
             'scale': (0.75, 1),
-            'num_colors': 10,
```

Comment:
The docstring of this function in `viewer_model.py` also uses `num_colors` in the example.

Reply:
Fixed in df05b49